criteria.json

```diff
@@ -0,0 +1,5 @@
+{
+    "high": 90,
```

Comment:
I think it will be good to add a comment here to indicate what those values mean.

Reply:
Thanks @jesus-serranop. This values are only demonstrative to be changed per customer needs. Also it would be a little problematic to have comments on the json itself, they will be taken as data.